### DISCUSSION TITLE
I need help with compiling the game...

### DISCUSSION THREAD

**Discussion by TripleLLL9** [FunkinCrew/Funkin#1707] [Compiling help] [answered]
Whenever I try to install polymod, discord_rpc and flixel-addons, here's what happens:
<img width="674" alt="Compiling Problem" src="https://user-images.githubusercontent.com/86983398/131253481-2485808b-d936-4aa8-b7dc-f568dfe3d5ee.PNG">

Any help?

**Comment by Cheemsandfriends** [comment]
Have you got Git-scm?

**Reply by TripleLLL9** [reply]
Yes I do.

**Comment by ninjamuffin99** [comment, marked as answer]
might be your git installation yeah

**Reply by TripleLLL9** [reply]
Actually it did compile after a couple of minutes, and turns out I had to update some libraries... But thank you for all of the help!
